Prototypes/BeleensMerge.cs

```diff
@@ -0,0 +1,497 @@
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    break;
+
+                case "Star Caster Staff":
+                    Adv.BuyItem("Castle", 48, req.Name);
+                    break;
+
+                case "Scarlet's Costume":
+                    if (!Core.IsMember)
+                    {
+                        Core.Logger($"{req.Name} requires Membership to Obtain");
+                        return;
+                    }
+                    Adv.BuyItem("Sleuthhound", 65, req.Name);
+                    break;
+
+                case "Infernal Dark Blade of Cruelty":
+                    if (!Core.IsMember)
+                    {
+                        Core.Logger($"{req.Name} requires Membership to Obtain");
+                        return;
+                    }
+                    Adv.BuyItem("Battleon", 10, req.Name);
+                    break;
+                case "Chaos Dragonlord Helm":
+                    Core.FarmingLogger($"{req.Name}", quant);
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("Stalagbite", "Vath", req.Name, isTemp: false);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    break;
+
+                case "Rose Aura of the Ascended":
+                    Core.Logger($"{req.Name} is seasonal AC item and only available Beleen's Birthday event.");
+                    Core.FarmingLogger($"{req.Name}", quant);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.Logger("This item is not setup yet"); //needs to be edited later as i couldn't find ShopID for it
+                    }
+                    break;
+
+                case "Prismatic Dye":
+                    Adv.BuyItem("Tower", 1966, req.Name);
+                    break;
+
+                case "Iron Dreadsaw":
+                    Core.FarmingLogger($"{req.Name}", quant);
+                    if (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        if (!Core.CheckInventory("Raw Dreadsaw"))
+                        {
+                            CoreNation.ApprovalAndFavor(10, 0);
+                            CoreNation.FarmDiamondofNulgath(5);
+                            CoreNation.SwindleBulk(10);
+                            Adv.BuyItem("ArchPortal", 1211, "Raw Dreadsaw");
+                        }
+                        CoreNation.ApprovalAndFavor(40, 20);
+                        CoreNation.FarmGemofNulgath(10);
+                        Adv.BuyItem("ArchPortal", 1211, req.Name);
+                    }
+                    break;
+
+                case "Taro's Manslayer":
```

Comment:
already exists ^_^ please use the pre-made function in: Nation\Various\TarosManslayer.cs
`public void GuardianTaro(bool ManslayerOnly = false)`

`TMS.GuardianTaro(true);`

--i added a non-mem bit for it that kills the dood for it